beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -959,6 +959,54 @@ func (s *Server) publishBlock(ctx context.Context, w http.ResponseWriter, r *htt
 	httputil.HandleError(w, "Body does not represent a valid block type", http.StatusBadRequest)
 }
 
+func (s *Server) PublishBlobs(w http.ResponseWriter, r *http.Request) {
```

Comment:
Echoing what @james-prysm said, this handler should be placed in `/home/radek/go/prysm/beacon-chain/rpc/prysm/beacon/handlers.go` since it's not an official Beacon API endpoint

Reply:
Updated